actions/generateComponent.js

```diff
@@ -16,7 +17,9 @@ function generateComponent(component, cmd, additionalOptions) {
       ...additionalOptions,
     },
   });
+  const linter = new Linter();
```

Comment:
I would move it below `componentInstance. generateComponent()` as you are not using it before component is created

Reply:
It doesn't matter. I think it's ok to create instances first, then operate on them